test/utility/Makefile.am

```diff
@@ -0,0 +1,5 @@
+AM_CPPFLAGS = -I$(top_builddir) -I$(top_srcdir)/src/include
+
+noinst_LTLIBRARIES = libtest_util.la
```

Comment:
@keithbostic Do you have an opinion about how we are best to add a ```test/utility``` directory that contains functions that are shared by several test programs?

I'd prefer not to:
* Have a single shared makefile in the test directory
* Have relative paths to files in Makefile.am
* Have a library file built for the content of the ```test/utility``` directory, and linked into test programs

Neither @daveh86 or I can see a solution that satisfies the above. Do you have any suggestions?